src/core/Match.pm

```diff
@@ -96,6 +96,19 @@ my class Match is Capture is Cool {
     }
 }
 
+multi sub infix:<eqv>(Match:D $a, Match:D $b) {
+    [&&] (
+	$a 	=:= $b,
```

Comment:
can `$a =:= $b` (container check, right?) be True while `$a.to` and `$b.to` are different?